templates/airtable/.nango/schema.ts

```
@@ -0,0 +1,86 @@
+
+export interface SyncMetadata_airtable_bases {
+};
+
+export interface Base {
+  id: string;
+  name: string;
+  permissionLevel: 'none' | 'read' | 'comment' | 'edit' | 'create';
+};
+
+/** @deprecated It is recommended to use a Model */
+export type ActionInput_airtable_whoami = void
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)